src/componets/DownVoting.jsx

```diff
@@ -0,0 +1,35 @@
+import { useState } from "react";
+
+export default function DownVoting({ singleArticle }) {
+  const [newVote, setNewVote] = useState(singleArticle.votes);
+  const [hasVoted, setHasVoted] = useState(false);
+
+  function handleClick() {
+    setHasVoted(false);
+    fetch(
+      `https://sobe-news.herokuapp.com/api/articles/${singleArticle.article_id}`
+    )
+      .then((res) => res.json())
+      .then(() => {
+        setNewVote(newVote - 1);
```

Comment:
in both of your requests you are not rendering this optimistically, its totally up to you if you want to do so, but if you wanted to you would need to call set state outside of the then block :)